deploy/osd-oauth-templates/ohss-2561/config.yaml

```
@@ -0,0 +1,3 @@
+deploymentMode: "Direct"
+direct:
+  environments: ["integration"]
```

This is an interesting case I didn't consider when setting up Direct.  The yaml for the "patch" doesn't apply if you use `oc apply -f` but you can apply the patch via the hive managing integration:

```suggestion
deploymentMode: "SelectorSyncSet"
selectorSyncSet:
  matchExpressions:
  - key: environment
    operator: In
    values: ["integration"]
  - key: ext-managed.openshift.io/hive-shard
    operator: In
    values: ["true"]
```